mlflow/data/huggingface_dataset.py

```diff
@@ -93,9 +93,10 @@ def ds(self) -> datasets.Dataset:
     @property
```

Comment:
Some fixes for HuggingFace while we're here

Reply:
thanks for adding these!